### DISCUSSION TITLE
Feature Request: Flame Graph

### DISCUSSION THREAD

**Discussion by hediet** [RecordReplay/devtools#4082] [Ideas]
A flame graph would be extremely useful to better understand which code-paths were executed:

![image](https://user-images.githubusercontent.com/2931520/138139228-088b617a-d1ed-483f-abe1-36e47d7db4ff.png)

Ideally, clicking on such an flame graph node would jump to the source and pause Replay's execution at that point.

**Comment by jasonLaster** [comment]
I personally cannot wait to build the call tree for the profiler. It is something that would be relatively easy for us to reconstruct while replaying and @gregtatum did a great job of designing a re-usable frontend.

re: clicking on a node on the call tree jumping to the execution point. Absolutely! That's our magical feature. We could also show the local variables on hover too :)
